src/pareto.rkt

```diff
@@ -1,9 +1,21 @@
 #lang racket
 
-(provide generate-pareto-curve (struct-out pareto-point) pareto-map pareto-union)
+(provide (struct-out pareto-point) pareto-map pareto-union pareto-minimize pareto-combine)
 
 (struct pareto-point (cost error data) #:prefab)
 
+(define (ppt->pt ppt)
+  (list (pareto-point-cost ppt) (pareto-point-error ppt)))
+  
+(define (pt->ppt pt)
+  (pareto-point (first pt) (second pt) (list)))
+
+(define (add-combinations ppt0 frontier)
+  (match-define (pareto-point cost0 err0 _) ppt0)
+  (for/list ([ppt (in-list frontier)])
+    (match-define (pareto-point cost err _) ppt)
+    (pareto-point (+ cost0 cost) (+ err0 err) (list))))
```

Comment:
Can we give this a more intuitive name? `pareto-offset` or `pareto-shift`? (Since it is shifting or offsetting the curve by a fixed amount?